scripts/exglobal_stage_ic.sh

```diff
@@ -72,6 +83,17 @@ for MEMDIR in "${MEMDIR_ARRAY[@]}"; do
       done
     done
   fi
+  
+  # Atmosphere Perturbation Files (usually used with replay ICS)
+  # Extra zero on MEMDIR ensure we have a number even if the string is empty
+  if (( 0${MEMDIR:3} > 0 )) && [[ "${USE_ATM_PERTURB_FILES:-false}" == "true" ]]; then
+      src="${BASE_CPLIC}/${CPL_OCNIC:-}/${PDY}${cyc}/${MEMDIR}/atmos/${PDY}.${cyc}0000.fv3_perturbation.nc"
```

Comment:
Why are atmosphere perturbations coming from the ocean IC directory? Doesn't matter right now since they are identical, but this should use `CPL_ATMIC`.

Reply:
Error in coping and pasting. Updated